src/System/PosixCompat/Files.hsc

```diff
@@ -132,6 +134,7 @@ import System.IO (IOMode(..), openFile, hFileSize, hSetFileSize, hClose)
 import System.IO.Error
 import System.PosixCompat.Types
 import System.Win32.File hiding (getFileType)
+import System.Win32.Time
```

Comment:
I know some of the other imports are not explicit, but would you mind being explicit here? I don't want to make things any worse than they already are.